presto-raptor/src/main/java/com/facebook/presto/raptor/storage/StorageManagerConfig.java

```diff
@@ -368,6 +369,20 @@ public StorageManagerConfig setMaxBufferSize(DataSize maxBufferSize)
         return this;
     }
 
+    @Min(1)
+    public int getMaxOpenFiles()
+    {
+        return maxOpenFiles;
+    }
+
+    @Config("storage.max-open-files")
```

Comment:
Maybe change to `"storage.max-created-files"` as we don't limit files opened for reads.